aoi.js documentation

Faf4a opened the pull request:
**Aoi.js Documentation Progress - Completed**

- Additions
> Completely finished all functions of all categories.
> Finished all guides.
> Double checked all functions, fixed things that were reported by users.
> Added `customfunction.md` and plenty of examples / going to improve later on.

- Changes
> Updated the aoi.panel guide with latest pictures and updated formatting.
> Fixed minor things such as formatting breaking etc.
> Redid whole structures of most guides, (most of it wont be shown on the website).
> Improved guides greatly (again).
>  \>- Updated `4status.md` and added sharding explanation to it.
>  \>- Added visual explanation in form of images to most guides.
>      \>- Added round corners to most images, perfection yk.
>  \>- Updated `aoi.music.md` to match other setups.
>  \>- Updated `introduction.md` to match other setups.
> Fixed formatting of all tables. Why? It annoyed me.

Everything should be fine as for now and updated to how it should be.

Leref commented:
Resolve conflicts.

Faf4a commented:
> Resolve conflicts.

Resolved.

Leref commented on src/functions/Awaited/awaitCmdReactions.md:
```diff
@@ -34,13 +34,13 @@ bot.command({
 
 bot.awaitedCommand({
-    name: "awaitedCommandExample",
-    code: `
```
Keep the format same.

Leref commented on src/functions/Awaited/reactionCollector.md:
```diff
@@ -24,24 +24,24 @@ $reactionCollector[channelID;messageID;userFilters;time;reactions;awaitedCommand
 });
 
 bot.awaitedCommand({
-    name: "awaitReaction",
```
Keep format same.

Leref commented on src/functions/Calling/broadcastEval.md:
````diff
@@ -26,9 +26,9 @@ This will return the amount of servers your bot is in:
 
 ```javascript
 bot.command({
-    name: 'broadcastEval',
````
Keep format same.

Leref commented on src/functions/Calling/clear.md:
````diff
@@ -27,8 +27,9 @@ This will delete the most recent fifty messages which are not pinned:
 
 ```javascript
 bot.command({
-    name: 'clear',
````
Keep format same.

Leref commented on src/functions/Calling/kick.md:
````diff
@@ -0,0 +1,36 @@
+```
+
+## Parameters 
+
````
Remove extra space.

Leref commented on src/functions/Calling/killShard.md:
````diff
@@ -0,0 +1,33 @@
+```
+
+## Parameters 
+
````
Remove extra space.

Leref commented on src/functions/Calling/messagePublish.md:
````diff
@@ -0,0 +1,35 @@
+```
+
+## Parameters 
+
````
Remove extra space

Leref commented on src/functions/Calling/modifyApplicationCommand.md:
````diff
@@ -0,0 +1,39 @@
+```
+
+## Parameters 
+
````
Remove extra space

Leref commented on src/functions/Calling/modifyApplicationCommand.md:
```diff
@@ -0,0 +1,39 @@
+| appID          | integer         | application command ID               |   true   |
+| ...options     | string          | new data for the application command |   true   |
+
+
```
Remove extra space

Leref commented on src/functions/Calling/modifyChannelPerms.md:
````diff
@@ -0,0 +1,44 @@
+```
+
+## Parameters 
+
````
Remove extra space

Leref commented on src/functions/Calling/modifyChannelPerms.md:
```diff
@@ -0,0 +1,44 @@
+    code: `
+  $modifyChannelPerms[$guildID;$channelID;+sendmessages;+addreactions]
+  `
+  // $guildID represents the @everyone role.
```
This should be placed somewhere else.

Leref commented on src/functions/Calling/modifyEmoji.md:
````diff
@@ -0,0 +1,38 @@
+```
+
+## Parameters 
+
````
Remove extra space

Leref commented on src/functions/Calling/modifyEmoji.md:
```diff
@@ -0,0 +1,38 @@
+| name      | string  | new emoji name                            |   true   |
+| ...roles? | integer | roles that will be able to use that emoji |  false   |
+
+
```
Remove double space

Leref commented on src/guides/6interaction.md:
```diff
@@ -2,7 +2,7 @@
 title: Interaction Commands
 description: This guide will be covering the basics about Slash Commands and their functionality. As well as everything
 else you might need to know.
-id: interactioncommands
```
Revert.

Leref commented on src/guides/8others.md:
```diff
@@ -93,12 +94,11 @@ const bot = new aoijs.AoiClient({
 
-In case this doesn't work, make sure you have a directory called "database", inside it a directory called "main" and a
-file called "main_scheme_1.sql" with the content of "{}".
+In case this doesn't work, make sure you have a directory called "database", inside it a directory called "main" and a file called "main_scheme_1.sql" with the content of "{}".
```
Rephrase this

Leref reviewed (changes requested):
All needed before approving.

Faf4a commented:
Everything resolved.

Faf4a commented:
Finished `calling`, continuing on `misc`.

- Updated a few descriptions.
- Fixed some spacing.
- Other minor stuff.

Faf4a commented:
**Aoi.js Documentation Progress - Completed**

- Additions
> Completely finished all functions of all categories.
> Finished all guides.
> Double checks all functions, fixed things that were reported by users.
> Added `customfunction.md` and plenty of examples.

- Changes
> Updated the aoi.panel guide with latest pictures and updated formatting.
> Fixed minor things such as formatting breaking etc.
> Redid whole structures of most guides, (most of it wont be shown on the website).
> Improved guides greatly (again).
> >- Updated `4status.md` and added sharding explanation to it.
> >- Added visual explanation in form of images to most guides.
>      >- Added round corners to most images, perfection yk.
> >- Updated `aoi.music.md` to match other setups.
> >- Updated `introduction.md` to match other setups.

Everything should be fine as for now and updated to how it should be.

- Fafa
